winterfell/webapp/src/js/claimSelectFormsCtrl.js

```diff
@@ -1,13 +1,65 @@
 var app = angular.module('vetafiApp');
-app.controller('claimSelectFormsCtrl', ['$scope', 'claimService', 'formTemplateService', '$stateParams',
-  function($scope, claimService, formTemplateService, $stateParams) {
+app.controller('claimSelectFormsCtrl', ['$scope', '$state', '$stateParams', 'claimService', 'formTemplateService',
+  function($scope, $state, $stateParams, claimService, formTemplateService) {
     $scope.claimId = $stateParams.claimId;
-    $scope.forms = formTemplateService;
+
+    $scope.allForms = formTemplateService; // returns an object {formId -> {...}}
+    var userForms = [ // TODO: state of all forms edited by user
```

Comment:
Ok great, naming is fine. And I think we're on the same page then. 
`formTemplateService` should continue to be used as an immutable list for all forms.

As for the state of current forms, we don't have it at the moment so for now, please let me keep this hardcoded variable so at least there's some UI present. When we're ready to hook things up in a future change, this hardcoded `userForms` will be gone. These last few changes  from me have been focusing on what we need for our MVP UI-wise anyway. There's a little more backend to be done anyway, but i want the hookup with the front-end to be simple.